pipelines/rj_iplanrio__sici/tasks.py

```
@@ -0,0 +1,95 @@
+        "Codigo_UA": "",
+        "Nivel": "",
+        "Tipo_Arvore": "",
+        "consumidor": consumidor,
+        "chaveAcesso": chave_acesso,
+    }
+
+
+@task
+def get_data_from_api_soap_sici(
+    wsdl: str = "http://sici.rio.rj.gov.br/Servico/WebServiceSICI.asmx?wsdl",
+    params: dict = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)